### PR TITLE
BookAwesome website 

### PR DESCRIPTION
**Pull request by codeWithHyder**
<h3>the book awesome</h3>

- Add mark up dynamicaly from from js file 
- form data saves in local storage

### CODE REVIEW

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @codeWithHyder, @byiringiroscar, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the README file doesn't have a future features section. Please kindly make changes.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    <ul class = 'booklst'>
+      <li>${bookList[i].title}</li>
+      <li> ${bookList[i].author}</li>
+        <button class ='btnr' type="button" onclick="removeBook(${i})">Remove</button>
+      </li>
+      <hr class='hr'>
+    </ul>
+    `;
+  }
+}
```
Comment (Munsa1) [inline script.js:25]: - [ ] As the user adds books to the `bookList` object, the number of books should increase in `locastorage` and on the page in the browser. Curently what happens is onlyt one book is being stored, so the latest book to be added is the only book showing in both the app and the browser.
image

File: script.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  <form class='booklst'>
+    <input type="text" name="title" id="booktitle" placeholder="Title">
+    <input type="text" name="author" id="bookauth" placeholder="Author">
+    <button type="button"  class='addbtn' onclick="addBook()">Add</button>
+  </form>
+`;
+formsection.appendChild(formSection);
+
+// function to add a new book to the list
+// eslint-disable-next-line no-unused-vars
```
Comment (Munsa1) [inline script.js]: - [ ] Kindly remove disable linters as linter confiurations help us to write ckean industry standard code.

### Note that this applies for all disable linters applied to this project

File: script.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  authorInput.value = '';
+  updateBookList();
+}
+
+// function to remove a book from the list
+// eslint-disable-next-line no-unused-vars
+function removeBook(index) {
+  bookList.splice(index, 1);
+  updateBookList();
+}
```
Comment (Munsa1) [inline script.js]: - [ ] Clicking the remove button should remove the selected book from both the app in the browser and from localstorage. Currently the book persists in localstorage after deletion. Kindly fix this

File: script.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  };
+  localStorage.setItem('storedData', JSON.stringify(info));
+});
+
+// get data from local storage
+const userObject = JSON.parse(localStorage.getItem('storedData'));
+if (userObject) {
+  bookTitle.value = userObject.title;
+  bookAuth.value = userObject.author;
+}
```
Comment (Munsa1) [inline script.js]: - [ ] Refreshing the page causes the book item to disapper from the app. The desired action should be that it should persist. To achieve this add an `onload` function which retrieves data from localstorage and displays it in the browser

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @codeWithHyder  @byiringiroscar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @codeWithHyder  @byiringiroscar , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approve

## Highlights
✅ Good job implementing the suggested changes

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codeWithHyder** [comment]
Thank you Amjad Brother

**Pull request merged**